server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java

```diff
@@ -153,27 +153,33 @@ private int getCorrespondingStatusCode(ResultStatus resultStatus) {
 
     private void logFailedCheck(Result result) {
         switch (result.getStatus()) {
-        case UNHEALTHY:
-            if (result.getError().isPresent()) {
-                LOGGER.error("HealthCheck failed for {} : {}",
-                    result.getComponentName().getName(),
-                    result.getCause().orElse(""),
-                    result.getError().get());
+            case UNHEALTHY:
+                if (result.getError().isPresent()) {
+                    LOGGER.error("HealthCheck failed for {} : {}",
+                        result.getComponentName().getName(),
+                        result.getCause().orElse(""),
```

Comment:
slf4j is doing null handling for you

Reply:
Is it handling optional for us?

Or are you suggesting: `result.getCause().orNull()` ? (hurts my eyes, I prefer the original version)

Reply:
ok